### PR TITLE
Feat/dex updates

### PR DESCRIPTION
**Pull request by jcompagni10**
Miscellaneous fixes to docs as well as updates for all v5 changes

### CODE REVIEW

File: docs/neutron/modules/dex/messages.md [hunk 1]
```diff
@@ -206,25 +218,25 @@ Taker limit orders will swap through all liquidity at ticks less than or equal t
...
 
 #### GOOD\_TIL\_CANCELLED
 
-Good-til-Cancelled limit orders are hybrid maker and taker limit orders. They will attempt to trade the supplied `AmountIn` at the `TickIndex` or better. However, if the total `AmountIn` cannot be traded at the limit price they remaining amount will be placed as a maker limit order. The proceeds from the taker portion are deposited into the user’s account immediately, however, the proceeds from the maker portion must be explicitly withdrawn via WithdrawLimitOrder.
+Good-til-Cancelled limit orders are hybrid maker and taker limit orders. They will attempt to trade the supplied `AmountIn` at the `LimitSellPrice` or better. However, if the total `AmountIn` cannot be traded at the limit price they remaining amount will be placed as a maker limit order. The proceeds from the taker portion are deposited into the user’s account immediately, however, the proceeds from the maker portion must be explicitly withdrawn via WithdrawLimitOrder.
```
Comment (NicholasDotSol) [inline docs/neutron/modules/dex/messages.md:239]:
```suggestion
Good-til-Cancelled limit orders are hybrid maker and taker limit orders. They will attempt to trade the supplied `AmountIn` at the `LimitSellPrice` or better. However, if the total `AmountIn` cannot be traded at the limit price the remaining amount will be placed as a maker limit order. The proceeds from the taker portion are deposited into the user’s account immediately, however, the proceeds from the maker portion must be explicitly withdrawn via WithdrawLimitOrder.
```

File: docs/neutron/modules/dex/messages.md [hunk 2]
```diff
@@ -289,14 +315,13 @@ enum LimitOrderType {
...
+| `MinAverageSellPrice` sdk.Dec          | Optional Minimum price that must be satisfied by the true output of a limit order                                                                           |
 
 ## Cancel Limit Order Message
 
 ### Overview
 
-Standard Taker limit orders (Good-til-cancelled & Good-til-Time) can be cancelled at any time if they have not been completely filled. Once a limit order is cancelled any remaining “TokenIn” liquidity is returned to the user.
-
-NOTE: Canceling a partially filled limit order does not withdraw the traded portion. A separate call must be made to `WithdrawFilledLimitOrder` to withdraw any proceeds from the limit order
+Standard Maker limit orders (Good-til-cancelled & Good-til-Time) can be cancelled at any (even if they have been filled). Once a limit order is cancelled any remaining `TokenIn` as well as `TokenOut` profits are returned to the `Creaator`.
```
Comment (NicholasDotSol) [inline docs/neutron/modules/dex/messages.md:324]:
```suggestion
Standard Maker limit orders (Good-til-cancelled & Good-til-Time) can be cancelled at any time (even if they have been filled). Once a limit order is cancelled any remaining `TokenIn` as well as `TokenOut` profits are returned to the `Creaator`.
```

File: docs/neutron/modules/dex/overview/concepts/tick-liquidity.md [hunk 3]
```diff
@@ -52,12 +51,14 @@ type TradePairID struct {
...
+When LP liquidity is deposited with a given fee and price it is added to the `TickLiquidity` instances such that the given fee is already included in the price.
+
+For example, if Alice deposits 100 TokenA  and 100TokenB at price 1 (tick 0) with a fee of 1 then both `PoolReserves` representing the `Pool` will be placed at tick  1 with a `MakerPrice` of 1.0001 each. If Bob were to swap 50Token0 for Token1 using Alice’s liquidity he would pay 1.0001 Token0 per Token1 (or an exchange rate of \~ .999 Token1 per Token0). His 50 Token0 would be deposited into the `Pool`'s `LowerTick0 PoolReserves`at tick 1 and fee 1. and he would receive 49 Token1 which would be deducted from  pool's `LowerTick1` `PoolReserves`.
```
Comment (NicholasDotSol) [inline docs/neutron/modules/dex/overview/concepts/tick-liquidity.md:61]:
```suggestion
For example, if Alice deposits 100 TokenA and 100TokenB at price 1 (tick 0) with a fee of 1 then both `PoolReserves` representing the `Pool` will be placed at tick 1 with a `MakerPrice` of 1.0001 each. If Bob were to swap 50 Token0 for Token1 using Alice’s liquidity he would pay 1.0001 Token0 per Token1 (or an exchange rate of \~ .999 Token1 per Token0). His 50 Token0 would be deposited into the `Pool`'s `LowerTick0 PoolReserves`at tick 1 and fee 1. and he would receive 49 Token1 which would be deducted from  pool's `LowerTick1` `PoolReserves`.
```

**Review by NicholasDotSol — APPROVED** [review]

File: static/img/dex_swap_pics.excalidraw
Comment (pr0n00gler) [inline static/img/dex_swap_pics.excalidraw:1]: Where is this file being used?

Comment (jcompagni10) [reply static/img/dex_swap_pics.excalidraw:1]: It's not. I wanted to add it to VC so its easier to update the images if we need to change them in the future

**Pull request merged**